ci/docker/aarch64-unknown-linux-musl/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM rust-aarch64-unknown-linux-musl
+
+ENV CC_aarch64_unknown_linux_musl=aarch64-unknown-linux-musl-gcc \
+    CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=aarch64-unknown-linux-musl-gcc
```

Comment:
So the question is, is this the name of the compiler inside the docker?

Reply:
Seems like it's aarch64-linux-musl, I'll update it